metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
With BC I was referring to doing the rename in Packetbeat. I think @brandonmensing was suggesting that `request_time` or `latency` might be more correct?

I agree that we want nanos on the long term, but we should definitely check how nanos display in Timelion and Kibana, humans are just not that good at parsing nanoseconds. For example, this is how a filter for transactions taking more than 1 second would look like: `responsetime > 1000000000`. And the legend on the Y axis takes half the space. And the compression in Elasticsearch works a lot worse.  Putting that in balance with having precision below milliseconds (who really cares?).. It seem like at least a debatable choice to me.